intermediate_source/flask_rest_api_tutorial.py

```diff
@@ -204,12 +207,12 @@ def get_prediction(image_bytes):
 # readable name.
 #
 # .. Note ::
-#    Did you notice why ``model`` variable is not part of ``get_prediction``
-#    method? Or why it is a global variable? Loading a model can be an
-#    expensive operation, memory, CPU wise. If we loaded the model in the
-#    ``get_prediction`` method, then it will get unnecessarily loaded every
+#    Did you notice that why ``model`` variable is not part of ``get_prediction``
```

Comment:
`why` still is incorrect. It should just be `Did you notice that model..`

Reply:
oops, somehow I missed it. Sent another PR #583 to fix it.